docker-compose.example.yml

```
@@ -1,51 +1,133 @@
-version: "3"
+version: '3.9'
 
 services:
-  db:
-    image: mariadb:10.5
-    restart: always
-    environment:
-      - MYSQL_DATABASE=photoview
-      - MYSQL_USER=photoview
-      - MYSQL_PASSWORD=photosecret
-      - MYSQL_RANDOM_ROOT_PASSWORD=1
-    volumes:
-      - db_data:/var/lib/mysql
-
   photoview:
-    image: viktorstrate/photoview:2
-    restart: always
+    image: viktorstrate/photoview:latest
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+      args:
+        ## Make sure to change values of the next 2 args if building on another platform type
+        BUILDPLATFORM: linux/amd64
+        TARGETPLATFORM: linux/amd64
+        #REACT_APP_API_ENDPOINT: http://localhost:4001/
+        UI_PUBLIC_URL: /
+        VERSION: selfbuilt
+        REACT_APP_BUILD_VERSION: selfbuilt
+        BUILD_DATE: $(date +%Y-%m-%d)
+        REACT_APP_BUILD_DATE: $(date +%Y-%m-%d)
+        COMMIT_SHA: $(git rev-parse --short HEAD)
+        REACT_APP_BUILD_COMMIT_SHA: $(git rev-parse --short HEAD)
```

This solved my issue [here](https://github.com/photoview/photoview/pull/910#issuecomment-1932482370). Thanks!

```suggestion
        BUILD_DATE: $$(date +%Y-%m-%d)
        REACT_APP_BUILD_DATE: $$(date +%Y-%m-%d)
        COMMIT_SHA: $$(git rev-parse --short HEAD)
        REACT_APP_BUILD_COMMIT_SHA: $$(git rev-parse --short HEAD)
```

---

Thanks, I tested it and it works for me. Have no idea why it was working with 1 $ before...
I've pushed this change to the repo